### PR TITLE
fix(widget): change type of ColumnSeries and colorSet

### PR DESCRIPTION
**Pull request by song2kim**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [X] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [X] `Error / Warning / Lint / Type`

### Description
Monthly Cost Widget
* change `XYSeries` type to `ColumnSeries`
* change using `colorSet`
* change tooltip

### CODE REVIEW

File: src/common/composables/amcharts5/index.ts [hunk 1]
```diff
@@ -108,7 +108,7 @@ export const useAmcharts5 = (
             if (!state.root) throw new Error('No root');
             return createXYLineSeries(state.root as Root, chart, settings);
         },
-        createXYColumnSeries: (chart: am5xy.XYChart, settings?: Partial<am5xy.IXYSeriesSettings>): am5xy.XYSeries => {
+        createXYColumnSeries: (chart: am5xy.XYChart, settings?: Partial<am5xy.IXYSeriesSettings>): am5xy.ColumnSeries => {
```
Comment (yuda110) [inline src/common/composables/amcharts5/index.ts:111]: praise: 😭

File: src/services/dashboards/widgets/monthly-cost/MonthlyCostWidget.vue [hunk 2]
```diff
@@ -168,34 +155,39 @@ const drawChart = (chartData: XYChartData[]) => {
     });
 
     const seriesSettings = {
-        name: 'usd_cost',
-        valueYField: 'usd_cost',
-        valueXField: 'date',
+        valueYField: 'date',
```
Comment (yuda110) [inline src/services/dashboards/widgets/monthly-cost/MonthlyCostWidget.vue:158]: It would be better make this constant! like `DATE_FIELD_NAME`?!

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/services/dashboards/widgets/monthly-cost/MonthlyCostWidget.vue (hunk 2 above)
Comment (song2kim) [reply src/services/dashboards/widgets/monthly-cost/MonthlyCostWidget.vue:158]: I think the field name is wrong while using the getRefinedXYChartData function, so I'll fix it! Thanx

**Review by WANZARGEN — APPROVED** [review]
LGTM~!!

**Pull request merged**